composer.json

```diff
@@ -19,7 +19,7 @@
         "phpunit/dbunit": "3.0.*",
         "facebook/webdriver" : "dev-master",
         "phpmd/phpmd": "^2.6",
-        "phan/phan": ">1.2.2"
+        "phan/phan": "~2.2.5"
```

Comment:
This is unnecessarily specific.. the composer.lock file pins the specific version for reproducibility (until the next time composer update is run).. I think it should stay a `>` operation.

Reply:
```suggestion
        "phan/phan": ">2.2.4"
```

Reply:
Done